src/rgw/rgw_lc.h

```diff
@@ -164,21 +206,26 @@ class LCRule
     dm_expiration = _dm_expiration;
   }
 
+  void set_filter(LCFilter* _filter){
```

Comment:
this setter is actually unused, since the only caller is from LCRule, I'll drop this